reactioncommerce/reaction
# refactor: move cart from server/no-meteor to node-app

Resolves #5591 
Impact: **minor**  
Type: **refactor**

As part of our effort to separate our Reaction Admin from our Reaction Node API, we are moving our GraphQL services from their respective `/imports/plugins/*/server/no-meteor/` folders into their new `/node-app/{core-services|plugins}/*` folder.

Updates also include future-proofing some existing code, including adding filetype extensions (and `index.js` to imports)

## Breaking changes
Any custom plugins that import files directly from the `cart` package will need to be updated.

## Testing
1. Start the application and see that it works as it should.
1. See that you can create a cart and complete an order
